game_eggs/steamcmd_servers/conan_exiles/egg-conan-exiles.json

```diff
@@ -1,31 +1,31 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1",
+        "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-01-20T12:47:23-05:00",
+    "exported_at": "2022-10-23T12:44:51+02:00",
     "name": "Conan Exiles",
     "author": "brycea@terrahost.cloud",
     "description": "Conan Exiles is an open-world survival game set in the brutal lands of Conan the Barbarian. Survive in a savage world, build your kingdom, and dominate your enemies in brutal combat and epic warfare.",
     "features": [
         "steam_disk_space"
     ],
-    "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
-    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_staging": "ghcr.io\/parkervcp\/yolks:wine_staging"
```

Comment:
Mayby add here something like Wine_staging|ghcr.io/parkervcp/yolks:wine_staging or just Wine|ghcr.io/parkervcp/yolks:wine_staging as there is no need for the end user to see the whole image. Or do you prefer that the see the whole image?

Reply:
i never changed the name xD

Reply:
but changed